Author: <cajus@debian.org>
Description: Fix location of configuration template.
Index: GOsa-2.7/gosa-core/include/functions.inc
===================================================================
--- GOsa-2.7.orig/gosa-core/include/functions.inc
+++ GOsa-2.7/gosa-core/include/functions.inc
@@ -44,7 +44,7 @@ if(!isset($_SERVER['CONFIG_FILE'])){
 }
 
 /* Define common locatitions */
-define ("CONFIG_TEMPLATE_DIR", "../contrib");
+define ("CONFIG_TEMPLATE_DIR", "/usr/share/doc/gosa");
 define ("TEMP_DIR","/var/cache/gosa/tmp");
 
 /* Define get_list flags */
