Author: <cajus@debian.org>
Description: Fix location of auto-generated class mapping file
Index: GOsa-2.7/gosa-core/include/class_config.inc
===================================================================
--- GOsa-2.7.orig/gosa-core/include/class_config.inc
+++ GOsa-2.7/gosa-core/include/class_config.inc
@@ -116,12 +116,12 @@ class config  {
            if we have installed or removed plugins. 
          */
         if(session::global_is_set("class_location.inc:timestamp")){
-            $tmp = stat("../include/class_location.inc");
+            $tmp = stat("/var/cache/gosa/class.cache");
             if($tmp['mtime'] != session::global_get("class_location.inc:timestamp")){
                 session::global_un_set("plist");
             }
         }
-        $tmp = stat("../include/class_location.inc");
+        $tmp = stat("/var/cache/gosa/class.cache");
         session::global_set("class_location.inc:timestamp",$tmp['mtime']);
 
         if($this->filename != "" && filemtime($this->filename) != $this->last_modified){
Index: GOsa-2.7/gosa-core/include/functions.inc
===================================================================
--- GOsa-2.7.orig/gosa-core/include/functions.inc
+++ GOsa-2.7/gosa-core/include/functions.inc
@@ -75,7 +75,7 @@ define('DES3_CBC_MD5',5);
 define('DES3_CBC_SHA1',16);
 
 /* Include required files */
-include_once("class_location.inc");
+require_once("/var/cache/gosa/class.cache");
 require_once ("functions_debug.inc");
 require_once ("accept-to-gettext.inc");
 
Index: GOsa-2.7/gosa-core/update-gosa
===================================================================
--- GOsa-2.7.orig/gosa-core/update-gosa
+++ GOsa-2.7/gosa-core/update-gosa
@@ -21,7 +21,7 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-define ("GOSA_HOME", dirname(__FILE__));
+define ("GOSA_HOME", "/usr/share/gosa");
 define ("LOCALE_DIR", GOSA_HOME."/locale");
 define ("PLUGSTATE_DIR", GOSA_HOME."/state");
 
@@ -138,8 +138,8 @@ function get_classes($folder= ".")
 function rescan_classes()
 {
 	echo "Updating class cache...\n";
-	$class_mapping= get_classes();
-	$filename= GOSA_HOME."/include/class_location.inc";
+	$class_mapping= get_classes("/usr/share/gosa");
+	$filename= "/var/cache/gosa/class.cache";
 
 	/* Sanity checks */
 	if (!file_exists($filename) || is_writable($filename)) {
